src/NexusMods.App.UI/Pages/Diagnostics/List/DiagnosticListView.axaml

```diff
@@ -10,69 +10,125 @@
+                                        <diagnostics:DiagnosticEntryView DataContext="{CompiledBinding .}" />
+                                    </DataTemplate>
+                                </ItemsControl.DataTemplates>
+                            </ItemsControl>  
+                        </ScrollViewer> 
+                    </TabItem>
+                    <TabItem Header="Warnings (4)" x:Name="WarningDiagnosticsTab">
+                        <ScrollViewer>
+                            <ItemsControl x:Name="WarningItemsControl">
+                                <ItemsControl.ItemsPanel>
+                                    <ItemsPanelTemplate>
+                                        <StackPanel Spacing="{StaticResource Spacing-2}"/>
+                                    </ItemsPanelTemplate>
+                                </ItemsControl.ItemsPanel>
+                                <!-- Do not replace with ReactiveUI, the generated component and kills shadows -->
+                                <!-- due to enforced ClipToBounds that can't be overwritten by styles -->
+                                <ItemsControl.DataTemplates>
+                                    <DataTemplate DataType="{x:Type diagnostics:IDiagnosticEntryViewModel}">
+                                        <diagnostics:DiagnosticEntryView DataContext="{CompiledBinding .}" />
+                                    </DataTemplate>
+                                </ItemsControl.DataTemplates>
+                            </ItemsControl>  
+                        </ScrollViewer> 
+                    </TabItem>
+                    <TabItem Header="Critical (6)" x:Name="CriticalDiagnosticsTab">
+                        <ScrollViewer>
+                            <ItemsControl x:Name="CriticalItemsControl">
+                                <ItemsControl.ItemsPanel>
+                                    <ItemsPanelTemplate>
+                                        <StackPanel Spacing="{StaticResource Spacing-2}"/>
+                                    </ItemsPanelTemplate>
+                                </ItemsControl.ItemsPanel>
+                                <!-- Do not replace with ReactiveUI, the generated component and kills shadows -->
+                                <!-- due to enforced ClipToBounds that can't be overwritten by styles -->
+                                <ItemsControl.DataTemplates>
+                                    <DataTemplate DataType="{x:Type diagnostics:IDiagnosticEntryViewModel}">
+                                        <diagnostics:DiagnosticEntryView DataContext="{CompiledBinding .}" />
+                                    </DataTemplate>
+                                </ItemsControl.DataTemplates>
+                            </ItemsControl>  
+                        </ScrollViewer> 
+                    </TabItem>
+                </TabControl>
```

Comment:
Remove the headers or replace them with an empty string. You can also move the body contents into a resource se we can re-use them.